setup.py

```diff
@@ -33,7 +33,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.1.0',
+    version='1.1.1',
```

Comment:
put 1.1.1.rc1 in case we find an issue in the install testing (rc is release candidate). If we find an issue and the version is already 1.1.1 we cannot upload another one to the test server easily.